### ISSUE TITLE
double schema.sql

### ISSUE THREAD

**Issue by u007** [gobuffalo/pop#66]
hi,

why is there 2 schema.sql? one on root directory and one in migration directory?
are these duplicate ? and we realize soda migrate seems to update migration directory schema.sql

which one is the actual one? can this be only single schema.sql?
it should be used for both schema load, dump, soda migrate.

and another question for soda,
soda drop / create / migrate seems to take environment variable GO_ENV,
but for soda schema load or schema dump, it seems to requires -e flag,
why isn't it picking up GO_ENV if -e flag is not specified?
and why soda create / drop take in parameter 1 as environment, but not able to specify -e flag?

and same goes for soda migrate...
```
export GO_ENV=test
soda drop
soda create
soda -e test schema load test
```

thank you

**Comment by stanislas-m** [comment]
@u007 schema.sql is only present in the migrations directory. Try to remove both files and migrate to see if you still have the problem.

**Comment by u007** [comment]
soda -e test schema load test seems to fail if schema.sql is missing from root

**Comment by stanislas-m** [comment]
My bad, it seems the defaults for `migrate` and `load` / `dump` schema.Sql location are different. I'll dig this a bit more.

**Comment by u007** [comment]
it turn out this accidental issue might be a bless somehow.
when we duplicate existing project, there are times when we remove all existing migration and only use schema load to load.

but eventually we add in new migration files before 1st deployment. and schema load should be done 1st, but if schema already contains all migrations, there will be error.
the solution to this is to insert schema migration manually to avoid the migration.

but i think its better to have only 1 schema as discussed above 👍

**Comment by stanislas-m** [comment]
Fixed with #133.

**Comment by u007** [comment]
thank you :)
